src/Stratis.Bitcoin.Features.BlockStore/BlockRepository.cs

```diff
@@ -12,26 +12,67 @@
 
 namespace Stratis.Bitcoin.Features.BlockStore
 {
+    /// <summary>
+    /// <see cref="IBlockRepository"/> is the interface to all the logics interacting with the blocks stored in the database.
+    /// </summary>
+    /// <seealso cref="System.IDisposable" />
     public interface IBlockRepository : IDisposable
     {
+        /// <summary>
+        /// Initializes the blockchain storage and ensure the genesis block has been created in the database.
+        /// </summary>
         Task InitializeAsync();
 
+        /// <summary>
+        /// Persist the next block hash and insert new blocks into the database.
+        /// </summary>
+        /// <param name="nextBlockHash">next block hash</param>
```

Comment:
We normally start with upper case and end with a `.`
>Next block hash.

Reply:
Thanks @dangershony  I will check the Commenting guides again and fix the issues.